.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+
+jobs:
+  test:
+    docker:
+      - image: circleci/golang:1.11.5
+    steps:
+      - checkout
+      - run:
+          name: License Test
+          command: make find-missing-license
+      - run: 
+          name: Go Tests
+          command: make test-go
+  release-test:
+    braches:
+      only:
+        - /^[0-9]\.[0-9]$/
+    docker:
+      - image: circleci/golang:1.11.5
+    steps:
+      - checkout
+      - run:
+          name: Version Test
+          command: make find-missing-version
+  deploy:
+    braches:
+      only:
+        - /^[0-9]\.[0-9]$/
+        - master
+    docker:
+      - image: circleci/golang:1.11.5
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run: make build-images
+      - run: make test-python
+      - run: make push-images 
+      - run: make build-and-upload-cli
+
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - test
+      - release-test:
+          filters:
+            branches:
+              only:
+                - /^[0-9]\.[0-9]$/
```

Comment:
This (and same for "deploy") seems like redundant config, since the branch filter is also defined on the "jobs". Is it necessary to be defined in both places?